editor/src/components/inspector/inpector-selectors.tsx

```diff
@@ -42,3 +44,16 @@ export const numberOfFlexContainersSelector = createSelector(
   selectedViewsSelector,
   numberOfFlexContainers,
 )
+
+export function useComputedSizeRef(prop: 'width' | 'height'): { readonly current: number | null } {
+  return useRefEditorState((store) => {
+    const metadata = metadataSelector(store)
+    const elementPath = selectedViewsSelector(store).at(0) ?? null
```

Comment:
We recently discovered that `.at()` is actually not as safe as we thought it was, because if the return type of `selectedViewsSelector(store)` is a `string` then `.at(0)` will just return the first character of that string. I know that seems completely nonsense in this specific case, but I'd much prefer us to use `[0]` because this caused a pretty nasty regression very recently when a return value was changed from Array to string